doc/devel/contributing.rst

```diff
@@ -296,24 +298,32 @@ besides which, we frequently underestimate how easy an issue is to solve!
 Contributing documentation
 ==========================
 
-Code is not the only way to contribute to Matplotlib. For instance,
-documentation is also a very important part of the project and often doesn't
-get as much attention as it deserves. If you find a typo in the documentation,
-or have made improvements, do not hesitate to send an email to the mailing
-list or submit a GitHub pull request.  To make a pull request, refer to the
-guidelines outlined in :ref:`how-to-contribute`.
+We are continuously striving to improve our documentation. You as an end-user
+of Matplotlib can a make valuable contribution to that because you will better
+see the potentials for improvement compared to a core developer. These include
+but are not limited to:
```

Comment:
Tried to do this directly like 15 times and get a 404, so:
> You as an end-user of Matplotlib can make a valuable contribution because you better (more clearly?) see the potentials for improvement as compared to a core developer.